### DISCUSSION TITLE
Google Ads: iOS conversions always at 0

### DISCUSSION THREAD

**Discussion by scamps86** [capawesome-team/capacitor-firebase#477] [Q&A]
Hi,

I'm encountering an issue with my Google Ads regarding the tracking conversions from iOS devices with Firebase Analytics.

Here's a brief overview of my problem:

**Event Tracking:** I've successfully set up event tracking for both iOS and Android in Google Analytics, and I can see the events from both platforms.
**Conversions Defined:** I have defined conversions for some of these events, and the tracking in Google Ads seems to work well for Android events.
**iOS Conversions Issue:** However, when it comes to iOS, I'm facing a peculiar problem. The conversions for iOS events are consistently showing as 0 in Google Ads, despite the events being tracked successfully in GA4.

Has anyone else encountered a similar issue with iOS conversions?
Are there specific considerations or steps that need to be taken for iOS configuration?
I appreciate any insights or suggestions you can provide to help me resolve this issue. Thanks in advance!

**Comment by robingenz** [comment]
Unfortunately, I don't have any experience with implementing Google Ads + Google Analytics. Maybe someone from the community has an idea. You could also post this question on the Ionic Discord server.

**Edit**: @scamps86 Maybe this helps you: https://support.google.com/google-ads/thread/43254168/campaign-showing-0-conversions?hl=en

**Reply by scamps86** [reply]
Thank you @robingenz, 

I've reviewed the post and the mobile conversion tracking article but I cannot detect a misconfiguration in my iOS app. :(
